src/FSharpPlus/Monoids.fs

```diff
@@ -15,7 +18,9 @@ module Dual = let run (Dual x) = x          : 'T
 /// The monoid of endomorphisms under composition.
 [<Struct; NoEquality; NoComparison>]
 type Endo<'t> = Endo of ('t -> 't) with
+    [<EditorBrowsable(EditorBrowsableState.Never)>]
```

Comment:
All the ``get_`` members I would leave them visible. No specific reasoning, simple because they are visible in standard F# classes.